src/codeintel/lib/codeintel2/tree_php.py

```diff
@@ -1185,6 +1185,10 @@ def _hits_from_first_part(self, tokens, scoperef):
                          first_token, scoperef)
                 return ([(elem, scoperef)], 1)
 
+        # Ignore keywords
+        if first_token in self.langintel.langinfo.keywords:
+            return (None, 1)
+
```

Comment:
I think you can move this to `eval_defns` and handle it like `eval_calltips` or at least WHERE calltips handles it.  If you can reuse `self.langintel.langinfo.keywords` that would be ideal.

Reply:
Wouldn't that be problematic because of the line above?
```python
if first_token in ("this", "self", "parent", "static"):
```

I'd have to allow those ones in `eval_defns`, which seems a little unclean.